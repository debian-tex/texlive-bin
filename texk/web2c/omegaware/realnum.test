#! /bin/sh

# Copyright (C) 2012 Peter Breitenlohner <tex-live@tug.org>
# You may freely use, modify and/or distribute this file.

tests=omegaware/tests

test -d $tests || mkdir -p $tests

TEXMFCNF=$srcdir/../kpathsea
OFMFONTS=./$tests
export TEXMFCNF OFMFONTS

./opl2ofm -verbose $srcdir/$tests/realnum $tests/xrealnum || exit 1

./ofm2opl -verbose xrealnum $tests/xrealnum || exit 1

diff $srcdir/$tests/realnum.out $tests/xrealnum.opl || exit 1

