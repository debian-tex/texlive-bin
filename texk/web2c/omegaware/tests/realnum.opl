(DESIGNSIZE R 10.0)
(CHARACTER C A
   (COMMENT no space, tab, or sign required after R)
   (CHARWD R)
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R+1)
   )
(CHARACTER C B
   (CHARWD R .)
   (CHARHT R 1.)
   (CHARDP R .1)
   (CHARIC R--1)
   )
