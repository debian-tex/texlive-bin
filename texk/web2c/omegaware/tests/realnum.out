(FAMILY UNSPECIFIED)
(FACE F MRR)
(CODINGSCHEME UNSPECIFIED)
(DESIGNSIZE R 10.0)
(COMMENT DESIGNSIZE IS IN POINTS)
(COMMENT OTHER SIZES ARE MULTIPLES OF DESIGNSIZE)
(CHECKSUM H 3996EA86)
(SEVENBITSAFEFLAG TRUE)
(CHARACTER H 41
   (CHARWD R 0.0)
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R 1.0)
   )
(CHARACTER H 42
   (CHARWD R 0.0)
   (CHARHT R 1.0)
   (CHARDP R 0.1)
   (CHARIC R 1.0)
   )
